packages/opencensus-instrumentation-grpc/src/grpc.ts

```diff
@@ -0,0 +1,478 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      gRPC://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {classes, logger, types} from '@opencensus/opencensus-core';
+import {EventEmitter} from 'events';
+import * as grpcModule from 'grpc';
+import * as lodash from 'lodash';
+import * as path from 'path';
+import * as semver from 'semver';
+import * as shimmer from 'shimmer';
+
+
+const findIndex = lodash.findIndex;
+
+function requireModule(module: string, nodeName: string) {
+  const indexPath = path.dirname(require.resolve(module));
+  return require(path.join(indexPath, nodeName));
+}
+const grpcClient = requireModule('grpc', 'src/client.js');
```

Comment:
We should avoid a dependency on the module being patched.

Reply:
PR #52